src/Helper/Scripts.php

```diff
@@ -1,68 +1,53 @@
 <?php
 /**
- *
  * This file is part of Aura for PHP.
  *
  * @license http://opensource.org/licenses/bsd-license.php BSD
- *
  */
 namespace Aura\Html\Helper;
 
 /**
- *
  * Helper for a stack of <script> tags.
  *
  * @package Aura.Html
- *
  */
 class Scripts extends AbstractSeries
 {
     private $capture;
 
     /**
-     *
```

Comment:
also looking at other aura repos I see there is an extra line in front for all. So to keep it consistent I would say not to remove these lines. Sorry for my earlier comment.